assoc.c

```diff
@@ -141,13 +140,11 @@ static void assoc_expand(void) {
 }
 
 void assoc_start_expand(uint64_t curr_items) {
-    if (started_expanding)
-        return;
-
-    if (curr_items > (hashsize(hashpower) * 3) / 2 &&
-          hashpower < HASHPOWER_MAX) {
-        started_expanding = true;
-        pthread_cond_signal(&maintenance_cond);
+    if (0 == pthread_mutex_trylock(&maintenance_lock)) {
```

Comment:
0 == ? is inconsistent with rest of codebase

Reply:
I apologize, an updated version is pushed that flips the comparison to the right hand side.